app/src/main/res/values/strings.xml

```diff
@@ -25,6 +25,11 @@
   <string name="state_learn_again_button">LEARN AGAIN</string>
   <string name="topic_overview_see_more">See More</string>
   <string name="topic_download_text">(%s MB)</string>
+  <string name="overview">Overview</string>
+  <string name="play">Play</string>
+  <string name="train">Train</string>
+  <string name="review">Review</string>
+  <string name="topic">Topic</string>
```

Comment:
This string appears to be unused--please remove.

Reply:
Removed